x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -933,7 +933,7 @@ public static void renderDocs() throws IOException {
     }
 
     protected static void renderDocs(String name) throws IOException {
-        if (System.getProperty("generateDocs") == null) {
+        if (System.getProperty("generateDocs") == null || true) {
```

Comment:
Add a comment about how we're turning these off please.

Reply:
Done